css/global.css

```diff
@@ -0,0 +1,8 @@
+#nav-flyout-ewc {
```

Comment:
This grabs the parent container holding the sidebar and hides it by overriding the existing CSS.